### ISSUE TITLE
Map selection is overly optimistic

### ISSUE THREAD

**Issue by ChrisKitching** [FAForever/client#372]
https://www.dropbox.com/s/bty7wgfifw1orp4/Screenshot%202016-06-08%2021.03.39.png?dl=0

**Comment by GrotheFAF** [comment]
some explanation that doesn't envolve dropbox?

**Comment by HardlySoftly** [comment]
Is that Windows 2000?

**Comment by Sheeo** [comment]
That's windows XP. Screenshot does explain itself; the map selection is suggesting to start a game with a .lnk (A shortcut file), supposedly?

**Comment by GrotheFAF** [comment]
No, it 's windows 7 in Windows Classic view. (see on the right network and desktop icons)

**Comment by Sheeo** [comment]
Right; forgot how archaic XP looks at this point.

**Comment by ChrisKitching** [comment]
It's Windows 7 with the theme engine and a few other pointless bits removed. A younger-me thought that was a good way to play video games and I never changed it yet. :P

But yes, to state the obvious: it seems to list all files in the map directory in the dropdown, including things that are blatantly not maps.

While this is perhaps not super unreaasonable on its own (people shouldn't be putting non-maps there by hand, after all), this explodes amusingly if you try and launch a game. We should at least do something to make sure we don't try and start a non-map

**Comment by ghost** [comment]
The image link is down.. I don't understand where it is, can someone make a screenshot or something ? Or is it out of date ?

**Comment by ChrisKitching** [comment]
The screenshot seems to be dead, but we don't need it. The bug was that, when creating a game, the list of maps you can choose from is just listing everything in the map directory. Even if they're not maps at all.
